lib/seccomp/CSystemCallFilter_Linux.cc

```diff
@@ -53,20 +53,21 @@ const struct sock_filter FILTER[] = {
     // Only applies to X86_64 arch. Jump to disallow for calls using the x32 ABI
     BPF_JUMP(BPF_JMP | BPF_JGT | BPF_K, UPPER_NR_LIMIT, 34, 0),
```

Comment:
This should change to 35 so it jumps to disallow. It's very easy to miss I'll make a PR either adding a comment or refactor the filter.